### PR TITLE
chore(site): make `info` gray

### PR DESCRIPTION
**Pull request by aslilac**
rather than having two yellows (`warning` and `notice`), I propose making `notice` the blue role rather than `info`, and make `info` a neutral gray.

### CODE REVIEW

**Comment by Parkreiner** [comment]
Doing some color auditing, and at the very least, it looks like we need to adjust the color usage for the workspace suggestion button

Screenshot 2024-08-20 at 9 47 59 AM
I think that removing the color here actually makes the link less obvious, and right now, the link is also failing WCAG minimum color contrast criteria (we need a minimum of 4.5:1)
Screenshot 2024-08-20 at 9 53 50 AM

I'm going to double-check the Storybooks

**Comment by Parkreiner** [comment]
@aslilac Do you know why the above wouldn't trigger a new Storybook? Can't tell if I'm missing something, but from what I can tell, the color wouldn't have changed until this PR

File: site/src/components/BuildAvatar/BuildAvatar.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const BuildAvatar: FC<BuildAvatarProps> = ({ build, size }) => {
 	const theme = useTheme();
 	const { status, type } = getDisplayWorkspaceBuildStatus(theme, build);
 	const badgeType = useClassName(
-		(css, theme) => css({ backgroundColor: theme.palette[type].light }),
+		(css, theme) => css({ backgroundColor: theme.roles[type].fill.solid }),
```
Comment (Parkreiner) [inline site/src/components/BuildAvatar/BuildAvatar.tsx:20]: Something I've kept forgetting to ask about: what is the difference between `palette` and `roles` again? Is one newer than the other, or are they meant to address different concerns?

File: site/src/theme/darkBlue/roles.ts [hunk 2]
```diff
@@ -72,6 +62,16 @@ export default {
...
+	info: {
+		background: colors.gray[950],
+		outline: colors.gray[400],
+		text: colors.gray[50],
+		fill: {
+			solid: colors.gray[500],
+			outline: colors.gray[600],
+			text: colors.white,
+		},
+	},
```
Comment (Parkreiner) [inline site/src/theme/darkBlue/roles.ts:74]: I know that Tailwind technically saturates their colors so the problem won't be as pronounced, but is gray on blue going to look washed out?

**Comment by aslilac** [comment]
> Do you know why the above wouldn't trigger a new Storybook? Storybook seems to have flagged virtually no changes

It did, I just approved all of the ones that I thought looked ok. I can keep the suggestions blue tho.

File: site/src/components/BuildAvatar/BuildAvatar.tsx (hunk 1 above)
Comment (aslilac) [reply site/src/components/BuildAvatar/BuildAvatar.tsx:20]: `roles` is supposed to replace `palette`. I'll add a deprecation note to `palette`. :)

File: site/src/theme/darkBlue/roles.ts (hunk 2 above)
Comment (aslilac) [reply site/src/theme/darkBlue/roles.ts:74]: I think they look fine, and we can always tweak it going forward.

**Review by buenos-nachos — APPROVED** [review]
Went through the stories, and I think the changes make a lot of sense!
The big thing I appreciate is that pending updates aren't as attention-grabby anymore

**Comment by aslilac** [comment]
> The big thing I appreciate is that pending updates aren't as attention-grabby anymore

I concur :^)

**Pull request merged**